tests/system_test.py

```diff
@@ -431,16 +432,19 @@ def error(msg):
             # If status is None, it means that the process is still
             # running. This is cool if expected is Process.RUNNING - we'll
             # check that below
-            self.terminate()  # send SIGTERM
-            try:
-                self.wait(TIMEOUT)
-            except TimeoutExpired:
-                self.send_signal(signal.SIGABRT)  # so that we may get core dump
-                error("did not terminate properly, required SIGABORT")
+            if self.abort_process:
+                self.send_signal(signal.SIGABRT)
```

Comment:
If I understand correctly the goal here is that there are some processes - like test servers, etc - that we really don't care if they are shutdown cleanly.

If that is the goal then I would recommend using SIGKILL rather than SIGABORT. Couple of reasons:

1. a program can mask sigabort. If the program is mis-behaving sending it a SIGABORT may not result in killing the program.
2. sigabort can generate a core dump. since the program being aborted is not something we're trying to debug the core dump just consumes resources

SIGKILL cannot be masked and it will force the process to exit immediately regardless of its state, which I think is what we want here.